addons/l10n_in/models/account_invoice.py

```diff
@@ -35,7 +35,14 @@ def _compute_amount_total_words(self):
     @api.depends('partner_id')
     def _compute_l10n_in_gst_treatment(self):
         for record in self:
-            record.l10n_in_gst_treatment = record.partner_id.l10n_in_gst_treatment
+            gst_treatment = record.partner_id.l10n_in_gst_treatment
```

Comment:
@jva-odoo There is something here that we would need to do from 16.0 and it is to not set this field if the company_id.fiscal_country_code, so the country_code field is not 'IN'.  It will be important for our own production.  (like it was done for state_id)  @odony

Reply:
oh, right we do it ASAP